backend/common/fbs/NetEncoding/Column.py

```diff
@@ -4,8 +4,9 @@
 
 import flatbuffers
```

Comment:
Do not reformat the contents of the `fbs/NetEncoding` directory - they are machine generated.